bodhi/server/consumers/pungi_masher.py

```diff
@@ -0,0 +1,522 @@
+                'syslinux-tftpboot',
+                'nosync',
+                '*-static',
+                'apitrace-libs',
+                'fakeroot-libs']
+        }
+        config['media_checksum_base_filename'] = '%(release_short)s-%(variant)s-%(version)s-%(arch)s-%(date)s%(type_suffix)s.%(respin)s'
+        config['multilib_blacklist'] = {
+            '*': [
+                'kernel',
+                'kernel-PAE*',
+                'kernel*debug*',
+                'dmraid-devel',
+                'kdeutils-devel',
+                'mkinitrd-devel',
+                'java-1.5.0-gcj-devel',
+                'java-1.7.0-icedtea-devel',
+                'php-devel',
+                'java-1.6.0-openjdk-devel',
+                'java-1.7.0-openjdk-devel',
+                'java-1.8.0-openjdk-devel',
+                'httpd-devel',
+                'tomcat-native',
+                'php*',
+                'httpd',
+                'krb5-server',
+                'krb5-server-ldap',
+                'mod_*',
+                'ghc-*'
+            ]
+        }
+        config['ostree'] = []
+        config['filter_system_release_packages'] = False
+        config['filter_packages'] = [
+            (
+                '(Workstation|Server|Cloud)$', {
+                    '*': [
+                        '^kernel.*debug.*',
+                        '^kernel-kdump.*',
+                        '^kernel-tools.*',
+                        '^syslog-ng.*',
+                        '^astronomy-bookmarks',
+                        '^generic.*',
+                        '^GConf2-dbus.*',
+                        '^bluez-gnome',
+                        '^java-1.8.0-openjdk',
+                        '^community-mysql.*',
+                        '^jruby.*',
+                        '^gimp-help-.*'
+                    ]
+                }
+            )
+        ]
+        config['pdc_insecure'] = False
+        config['variants_file'] = 'variants-base-runtime.xml'
+        config['createiso_skip'] = [
+            (
+                '^base-runtime$', {
+                    'src': True,
+                    '*': True
+                }
+            ), (
+                '^Server$', {
+                    'src': True
+                }
+            ), (
+                '^Cloud$', {
+                    'src': True,
+                    '*': True
+                }
+            ), (
+                '^Everything$', {
+                    'src': True,
+                    '*': True
+                }
+            )
+        ]
+        config['gather_method'] = 'nodeps'
+        config['bootable'] = True
+        config['gather_source'] = 'module'
+        config['pkgset_koji_tag'] = 'unknown'
+        config['global_target'] = 'f25'
+        config['release_is_layered'] = False
+        config['release_name'] = 'Test'
+        config['greedy_method'] = 'build'
+        return config
+
+    def validate(cls):
+        pass
```

Comment:
We discussed this last week, but just to formally note: we agreed to move most of this code into a library inside Pungi with a documented public API. Then we can adjust Bodhi to use that new public API to drive this code.